src/schema/v2/conversation/conversations.ts

```diff
@@ -1,13 +1,43 @@
-import { pageable } from "relay-cursor-paging"
+import { CursorPageable, pageable } from "relay-cursor-paging"
 import { convertConnectionArgsToGravityArgs } from "lib/helpers"
-import { GraphQLInt, GraphQLFieldConfig } from "graphql"
+import {
+  GraphQLInt,
+  GraphQLFieldConfig,
+  GraphQLString,
+  GraphQLBoolean,
+} from "graphql"
 import { connectionFromArraySlice, connectionDefinitions } from "graphql-relay"
 import { assign } from "lodash"
 
 import { ConversationType } from "schema/v2/conversation"
 import { ResolverContext } from "types/graphql"
+import { GraphQLEnumType } from "graphql"
 
-const Conversations: GraphQLFieldConfig<void, ResolverContext> = {
+interface ConversationsArguments extends CursorPageable {
+  dismissed?: boolean
+  hasMessage?: boolean
+  hasReply?: boolean
+  partnerId?: string
+  type?: "Partner" | "User"
+}
+
+const ConversationsTypeEnum = new GraphQLEnumType({
+  name: "ConversationType",
```

Comment:
#superminor this isn't really a conversation 'type' (which is a generic-y name), is it? it's more 'ConversationsInputMode' or something (for these kinds of synthetic-y types that are not likely to be reused and refer to some input argument (as in this case), i wonder if its better to be explicit about that in their naming).

Reply:
Will update 👍